src/QtComponents/QtTopologyVertexCreationAction.cpp

```diff
@@ -10,11 +10,15 @@
 #include <QtUtil/QtErrorManagement.h>
 #include "QtComponents/QtMgx3DMainWindow.h"
 #include "QtComponents/QtTopologyVertexCreationAction.h"
+#include "QtComponents/QtNumericFieldsFactory.h"
 
 #include <TkUtil/MemoryError.h>
 #include <TkUtil/InternalError.h>
 #include <QtUtil/QtConfiguration.h>
 #include <QtUtil/QtUnicodeHelper.h>
+#include "Geom/Vertex.h"
+
+#include <values.h>
```

Comment:
```suggestion
#include <climits>
```
The `values.h` header is deprecated